### ISSUE TITLE
Is it possible to make class names for xorg?

### ISSUE THREAD

**Issue by ShadiestGoat** [hajimehoshi/ebiten#2130]
Hi!
I'm building my game on a linux system which uses xorg. My system is using i3-gaps, a tiling window manager, however, this game is best debugged while its in a floating state. If I set a default state for applications, I usually set it using their class, because most applications have a unique class, right? But I can't seem to figure out how to set this class using ebiten. 
The way I test a window's class is using `xprop WM_CLASS` if thats of any use.

**Comment by hajimehoshi** [comment]
Sorry but I'm not so familiar with X Window and class names for them. Would an API to get a native window handler be fine?

**Comment by ShadiestGoat** [comment]
I'm not sure. Probably would though.

**Comment by hajimehoshi** [comment]
> Would that be different based on GOOS?

Yes. So the API would return a uinptr. For Windows, this would be HWND, for example.

See also
* https://pkg.go.dev/github.com/go-gl/glfw/v3.3/glfw#Window.GetX11Window
* https://pkg.go.dev/github.com/go-gl/glfw/v3.3/glfw?GOOS=windows#Window.GetWin32Window
* https://pkg.go.dev/github.com/go-gl/glfw/v3.3/glfw?GOOS=darwin#Window.GetCocoaWindow

**Comment by ShadiestGoat** [comment]
Wait, why not just get access to a `*glfw.Window`? That would easily allow for further expansion, right?

**Comment by hajimehoshi** [comment]
It's because the implementation has been changing. Actually, in the latest version for Windows, Ebitengine has an original implementation.
